src/listener/hastebinConversion.ts

```diff
@@ -44,7 +44,8 @@ export default class HastebinConversionListener extends Listener {
 			)!
 		);
 		const embed = new MessageEmbed()
-			.setTitle(hastebinUrl)
+			.setTitle('Converted!')
+			.setDescription(`A message attachment from ${msg.author.tag} has been converted to a hastebin post.\nYou can view it __[here](${hastebinUrl})__.`)
```

Comment:
I think having it like this will make it just a tiny bit less spammy.

```suggestion
			.setDescription(`__[View ${messageAttachment.name || "attachment"} on hastebin](${hastebinUrl})__.`)
```